torch/_inductor/ir.py

```diff
@@ -6044,15 +6044,26 @@ def __init__(  # type: ignore[no-untyped-def]
         *,
         unbacked_bindings=None,
     ) -> None:
+        # When aten binary ops have constant second args, cpp wrapper expects the scalar
+        # version.  This should long-term be handled as in
+        # https://github.com/pytorch/pytorch/issues/90923.
+        BINARY_OP_MAPPING = {
+            aten.add.Tensor: aten.add.Scalar,
+            aten.div.Tensor: aten.div.Scalar,
+            aten.divide.Tensor: aten.divide.Scalar,
+            aten.floor_divide: aten.floor_divide.Scalar,
```

Comment:
Note for reviewers: this is intentional; `floor_divide`'s Tensor operator has no suffix.